primitives/io/src/lib.rs

```diff
@@ -28,6 +28,9 @@
 
 use sp_std::vec::Vec;
 
+#[cfg(feature = "std")]
+use futures::executor::block_on;
```

Comment:
Is it ok to totally block a runtime execution on potentially some long-duration I/O?